scripts/create-launchpad-credentials

```diff
@@ -39,19 +41,25 @@ class AuthorizeRequestTokenRemotely(RequestTokenAuthorizationEngine):
 
 
 def main():
+    parser = ArgumentParser()
+    parser.add_argument(
+        "-l", "--launchpad", dest="launchpad_instance", default="production")
```

Comment:
What are the values it accepts?

I see "production" as default, so I also assume "staging", but for local instances it does it by IP?